kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/food2door/Food2DoorApp.java

```diff
@@ -0,0 +1,27 @@
+package com.kodilla.good.patterns.food2door;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Food2DoorApp {
+    public static void main(String[] args) {
```

Comment:
Please create class Food2DoorShop, with methods:
placeOrder(order)
setSuppliers()

placeOrder will call process on proper Supplier (for example extra food shop)